src/swift/fila.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  fila.swift
+//
+//  Created by Matheus Torres on 14/11/20.
+//
+
+class Queue {
+    var queue: [Int] = []
+    
+    func enqueue(_ value: Int) {
+        self.queue.append(value)
+    }
+    
+    func dequeue(at indexPath: Int) {
+        self.queue.remove(at: indexPath)
```

Comment:
Como o conceito de fila segue o padrão FIFO (first in, first out), ou seja, o primeiro a entrar é o primeiro a sair, o ideal seria o método `deque` não receber argumentos, ele deveria simplesmente remover e retornar o primeiro elemento de `queue`.